module/BeadPoolManifest.py

```diff
@@ -118,6 +118,8 @@ def __parse_file(self, manifest_file):
             for locus_idx in xrange(self.num_loci):
                 self.normalization_ids[locus_idx] += 100 * \
                     self.assay_types[locus_idx]
+                # To mimic the byte-wrapping behavior from GenomeStudio, AutoCall, IAAP take the mod of 256
+                self.normalization_ids[locus_idx] %= 256
```

Comment:
I am guessing that the bug was not in develop?

Reply:
The behavior @1teaspoon observed is reproducible in develop, not related to this branch.